codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -991,7 +998,7 @@ public void write(Object msg, final ChannelPromise promise) {
                     return;
                 }
 
-                if (!isActive()) {
+                if (!isActive() || outboundClosed) {
```

Comment:
Same comment as above. We may not want to write "headers"/"data", but other control type frames may still be valid based upon the current usage of `outboundClosed`.

Reply:
yeah let us just fail headers / data to be inline with the RFC.